openedx/core/djangoapps/user_api/tests/test_constants.py

```diff
@@ -1,254 +1,254 @@
+    (u'NR', u'Nauru'),
+    (u'NP', u'Nepal'),
+    (u'NL', u'Netherlands'),
+    (u'NC', u'New Caledonia'),
+    (u'NZ', u'New Zealand'),
+    (u'NI', u'Nicaragua'),
+    (u'NE', u'Niger'),
+    (u'NG', u'Nigeria'),
+    (u'NU', u'Niue'),
+    (u'NF', u'Norfolk Island'),
+    (u'KP', u'North Korea'),
+    (u'MP', u'Northern Mariana Islands'),
+    (u'NO', u'Norway'),
+    (u'OM', u'Oman'),
+    (u'PK', u'Pakistan'),
+    (u'PW', u'Palau'),
+    (u'PS', u'Palestine, State of'),
+    (u'PA', u'Panama'),
+    (u'PG', u'Papua New Guinea'),
+    (u'PY', u'Paraguay'),
+    (u'PE', u'Peru'),
+    (u'PH', u'Philippines'),
+    (u'PN', u'Pitcairn'),
+    (u'PL', u'Poland'),
+    (u'PT', u'Portugal'),
+    (u'PR', u'Puerto Rico'),
+    (u'QA', u'Qatar'),
+    (u'RE', u'R\xe9union'),
+    (u'RO', u'Romania'),
+    (u'RU', u'Russia'),
+    (u'RW', u'Rwanda'),
+    (u'BL', u'Saint Barth\xe9lemy'),
+    (u'SH', u'Saint Helena, Ascension and Tristan da Cunha'),
+    (u'KN', u'Saint Kitts and Nevis'),
+    (u'LC', u'Saint Lucia'),
+    (u'MF', u'Saint Martin (French part)'),
+    (u'PM', u'Saint Pierre and Miquelon'),
+    (u'VC', u'Saint Vincent and the Grenadines'),
+    (u'WS', u'Samoa'),
+    (u'SM', u'San Marino'),
+    (u'ST', u'Sao Tome and Principe'),
+    (u'SA', u'Saudi Arabia'),
+    (u'SN', u'Senegal'),
+    (u'RS', u'Serbia'),
+    (u'SC', u'Seychelles'),
+    (u'SL', u'Sierra Leone'),
+    (u'SG', u'Singapore'),
+    (u'SX', u'Sint Maarten (Dutch part)'),
+    (u'SK', u'Slovakia'),
+    (u'SI', u'Slovenia'),
+    (u'SB', u'Solomon Islands'),
+    (u'SO', u'Somalia'),
+    (u'ZA', u'South Africa'),
+    (u'GS', u'South Georgia and the South Sandwich Islands'),
+    (u'KR', u'South Korea'),
+    (u'SS', u'South Sudan'),
+    (u'ES', u'Spain'),
+    (u'LK', u'Sri Lanka'),
+    (u'SD', u'Sudan'),
+    (u'SR', u'Suriname'),
+    (u'SJ', u'Svalbard and Jan Mayen'),
+    (u'SZ', u'Swaziland'),
+    (u'SE', u'Sweden'),
+    (u'CH', u'Switzerland'),
+    (u'SY', u'Syria'),
+    (u'TW', u'Taiwan'),
+    (u'TJ', u'Tajikistan'),
+    (u'TZ', u'Tanzania'),
+    (u'TH', u'Thailand'),
+    (u'TL', u'Timor-Leste'),
+    (u'TG', u'Togo'),
+    (u'TK', u'Tokelau'),
+    (u'TO', u'Tonga'),
+    (u'TT', u'Trinidad and Tobago'),
+    (u'TN', u'Tunisia'),
+    (u'TR', u'Turkey'),
+    (u'TM', u'Turkmenistan'),
+    (u'TC', u'Turks and Caicos Islands'),
+    (u'TV', u'Tuvalu'),
+    (u'UG', u'Uganda'),
+    (u'UA', u'Ukraine'),
+    (u'AE', u'United Arab Emirates'),
+    (u'GB', u'United Kingdom'),
+    (u'UM', u'United States Minor Outlying Islands'),
+    (u'US', u'United States of America'),
+    (u'UY', u'Uruguay'),
+    (u'UZ', u'Uzbekistan'),
+    (u'VU', u'Vanuatu'),
+    (u'VE', u'Venezuela'),
+    (u'VN', u'Vietnam'),
+    (u'VG', u'Virgin Islands (British)'),
+    (u'VI', u'Virgin Islands (U.S.)'),
+    (u'WF', u'Wallis and Futuna'),
+    (u'EH', u'Western Sahara'),
+    (u'YE', u'Yemen'),
+    (u'ZM', u'Zambia'),
+    (u'ZW', u'Zimbabwe')
```

Comment:
Behold the single quoted glory!

Reply:
Darn - I should have stayed with the double quotes to show the actual changes that occurred in the file. Guess countries were renamed/added in the new django-countries version.

Reply:
I changed the quotes back to double quotes so the changes could be seen.